src/components/config/channels/whatsapp/components/tabs/ProfileTab.vue

```diff
@@ -285,16 +289,31 @@
             return;
           }
 
-          const payload = {
+          let payload = this.removeEmptyProperties({
```

Comment:
doesn't the imported `removeEmpty` method do the same as this `removeEmptyProperties`? why have a local method for this?